packages/@azure/servicebus/data-plane/test/testUtils.ts

```
@@ -40,13 +42,50 @@ export enum ClientType {
   TopicFilterTestSubscription
 }
 
-export function getSenderClient(
+export async function getSenderClient(
   namespace: Namespace,
   clientType: ClientType
-): QueueClient | TopicClient {
+): Promise<QueueClient | TopicClient> {
   switch (clientType) {
     case ClientType.PartitionedQueue:
-      return namespace.createQueueClient(process.env.QUEUE_NAME || "partitioned-queue");
+      const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"] || "";
```

Lets throw an error any of the environment variables is not available